int8 device
---
string status